### PR TITLE
Add notes on run tasks token permissions

### PR DESCRIPTION
**Pull request by cam-stitt**
### What
Added a note about the run tasks access token and the permissions it allows.

https://www.terraform.io/cloud-docs/users-teams-organizations/permissions#permissions-outside-terraform-cloud-s-scope

### Why
This is an additional security concern that customers need to be aware of when using run tasks.

----------

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
- [ ] Description links to related pull requests or issues, if any.

#### Content
- [ ] Redirects have been added for moved, renamed, or deleted pages.
- [ ] Sidebar navigation files have been updated for added, deleted, reordered, or renamed pages.
- [ ] Links to related content where appropriate (e.g., API endpoints, permissions, etc.).
- [ ] Pages with related content are updated and link to this content when appropriate.
- [ ] New pages have metadata (page name and description) at the top.
- [ ] New images are 2048 px wide. They have HashiCorp standard annotation color (#F92672) and format (rectangle with rounded corners), blurred sensitive details (e.g. credentials, usernames, user icons), and descriptive alt text in the markdown for accessibility.
- [ ] New code blocks have the correct syntax and line breaks to eliminate horizontal scroll bars.
- [ ] UI elements (button names, page names, etc.) are bolded.
- [x] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by glennsarti — APPROVED** [review]
Approving.  My comment should not block merging.

File: content/cloud-docs/users-teams-organizations/permissions.mdx [hunk 1]
```diff
@@ -201,5 +201,8 @@ Since Terraform Cloud integrates with other systems, the permissions models of t
 
 - When a workspace is connected to a VCS repository, anyone who can merge changes to that repository's main branch can indirectly queue plans in that workspace, regardless of whether they have explicit permission to queue plans or are even a member of your Terraform Cloud organization. (And when auto-apply is enabled, merging changes will indirectly apply runs.)
 - If you use Terraform Cloud's API to create a Slack bot for provisioning infrastructure, anyone able to issue commands to that Slack bot can implicitly act with that bot's permissions, regardless of their own membership and permissions in the Terraform Cloud organization.
+- When a run task sends a request to an integrator, it provides an access token that provides access depending on the run task stage:
+  - For post-plan, it provides access to the runs plan json and the run task callback
+  - All access tokens created for run tasks have a lifetime of 10 minutes
```
Comment (glennsarti) [inline content/cloud-docs/users-teams-organizations/permissions.mdx:206]: Just thinking;

Is it possible to be less specific about the `10 minutes` Just so we don't have to remember to change this document if we change the underlying code. e.g. `- All access tokens created for a task are short-lived and are only valid while the task is running`

Comment (cam-stitt) [reply content/cloud-docs/users-teams-organizations/permissions.mdx:206]: I did think about that, but the access token has a TTL that is defined separately to the run task timeout. Therefore technically, we could set a different time to the access token if that was something we needed.

Comment (glennsarti) [reply content/cloud-docs/users-teams-organizations/permissions.mdx:206]: Fair enough.  That's raised another question in my head, but that's for the TFC repo ..

**Review by jarrodn-au — APPROVED** [review]

**Review by laurapacilio — APPROVED** [review]

**Pull request merged**